### PR TITLE
fixed row and column subtract button values for issue #755

### PR DESCRIPTION
**Pull request by NARUDESIGNS**
Fixed issue with the row and column subtract buttons. These buttons when clicked previously yielded a negative text value in the button. However, with the new fix, the text values shouldn't be any less than 1 row and 1 column as a table generally should have at least 1 row and 1 column.
Please help review and feedback @TildaDares @jywarren 
Thank you.

### CODE REVIEW

**Review by TildaDares — CHANGES_REQUESTED** [review]
Hi @NARUDESIGNS, your formatter/linter has added more changes than required. Do you think you could disable your linter and then undo the changes it added? Thank you!

File: src/modules/PublicLab.RichTextModule.Table.js [hunk 1]
```diff
@@ -12,62 +12,70 @@ module.exports = function initTables(_module, wysiwyg) {
...
 
-
   $(document).on('click', '#incRows', function() {
-    $("#tableRows").text( Number($("#tableRows").text()) + 1 );
+    $('#tableRows').text(Number($('#tableRows').text()) + 1);
   });
   $(document).on('click', '#decRows', function() {
-    $("#tableRows").text( Number($("#tableRows").text()) - 1 );
+    if (Number($('#tableRows').text()) > 1) {
+      $('#tableRows').text(Number($('#tableRows').text()) - 1);
```
Comment (TildaDares) [inline src/modules/PublicLab.RichTextModule.Table.js:69]: I think the `Number($('#tableRows').text())` value should be assigned to a variable so that there's less repetition. The same thing applies to lines 76-77. Thanks!

Also, you need to run `grunt build` to compile into `/dist/PublicLab.Editor.js`

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: src/modules/PublicLab.RichTextModule.Table.js (hunk 1 above)
Comment (NARUDESIGNS) [reply src/modules/PublicLab.RichTextModule.Table.js:69]: OK thank you. Do I have to push the dist/PublicLab.Editor.js changes as well?
I only pushed the file where I added those `if`conditions...

**Comment by NARUDESIGNS** [comment]
> Hi @NARUDESIGNS, your formatter/linter has added more changes than required. Do you think you could disable your linter and then undo the changes it added? Thank you!

OK I will, thanks for pointing out.

File: src/modules/PublicLab.RichTextModule.Table.js (hunk 1 above)
Comment (TildaDares) [reply src/modules/PublicLab.RichTextModule.Table.js:69]: Yes you also have to push the changes in the ‘dist’ folder.

**Comment by NARUDESIGNS** [comment]
@TildaDares I've reverted the changes, disabled prettier (my formatter), created a variable and made the changes. Please review and feedback. 
Thank you so so much.
Happy weekend!

File: dist/PublicLab.Editor.js [hunk 2]
```diff
@@ -22475,7 +22475,7 @@ module.exports = function(textarea, _editor, _module) {
     );
     $(".wk-commands, .wk-switchboard").addClass("btn-group");
     $(".wk-commands button, .wk-switchboard button").addClass(
-        "btn btn-light"
+        "btn btn-outline-secondary"
```
Comment (TildaDares) [inline dist/PublicLab.Editor.js:22478]: Hi @NARUDESIGNS, is this change related to this PR?

Everything else looks great btw. Great job!

Comment (NARUDESIGNS) [reply dist/PublicLab.Editor.js:22478]: @TildaDares this change happens automatically when I work on src/modules/PublicLab.RichTextModule.Table.js and I don't know why. Do I need to take it out manually?

Comment (TildaDares) [reply dist/PublicLab.Editor.js:22478]: @NARUDESIGNS Undo the changes and then do a push. Let's see if that works.

Comment (NARUDESIGNS) [reply dist/PublicLab.Editor.js:22478]: I should undo the changes just for dist/Publiclab.Editor.js right?

Comment (NARUDESIGNS) [reply dist/PublicLab.Editor.js:22478]: > @TildaDares this change happens automatically when I work on src/modules/PublicLab.RichTextModule.Table.js and I don't know why. Do I need to take it out manually?

Also just to update you, this automatic change (addition of "btn btn-outline-secondary" property) in the dist/Publiclab.Editor.js happens in my local env just before I push to remote repo this is why I was asking you if I need to manually remove the change.
@TildaDares

Comment (TildaDares) [reply dist/PublicLab.Editor.js:22478]: Hi @NARUDESIGNS, I'm not sure why but that change is added every time you run `grunt build`.

Comment (TildaDares) [reply dist/PublicLab.Editor.js:22478]: The problem is with this commit https://github.com/publiclab/PublicLab.Editor/commit/b1cb34626b1c3acf8a9d87f12c4ee4830d5f8471. `grunt build` was supposed to be run after adding the change so every time anyone else runs `grunt build`, it adds that change to the `dist` folder.

Leave the change in one of your PRs and remove it from the other. All you have to do is undo just that line of change after running  `grunt build`.

Comment (NARUDESIGNS) [reply dist/PublicLab.Editor.js:22478]: Ok, thank you very much. I have removed it after running `grunt build` for PR https://github.com/publiclab/PublicLab.Editor/pull/758 while the change remains here on PR https://github.com/publiclab/PublicLab.Editor/pull/757 just like you advised.
Please review, thank you so much for your time @TildaDares

**Review by TildaDares — APPROVED** [review]
Great job @NARUDESIGNS. LGTM!

Could you add a screen recording of the change just to make sure everything works as it should? Thank you!

**Comment by NARUDESIGNS** [comment]
Sure, not a problem. I will!
Thank you for your time.

**Comment by NARUDESIGNS** [comment]
Hi, @TildaDares here's a short screen record to show you how it behaves now. 
With the resolution, the number of rows or cols can not be less than 1. 
However, I'd like your opinion on whether we need to set max rows and cols as well (which I think we should do).
Thank you.

https://user-images.githubusercontent.com/49737323/142673593-e6ddf8d4-10cf-4f06-8327-2ca679a69eef.mov

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
This one is nice and simple, so let's merge it! It's not complex enough to need a test. Thanks!

**Comment by NARUDESIGNS** [comment]
Merged!
Thank you!!!!
@TildaDares @jywarren
